src/types.db

```diff
@@ -78,6 +78,9 @@ frequency_offset	value:GAUGE:-1000000:1000000
 frequency		value:GAUGE:0:U
 fscache_stat		value:DERIVE:0:U
 gauge			value:GAUGE:U:U
+gps_sat			value:GAUGE:0:25
+gps_hdop		value:GAUGE:0:100
```

Comment:
Can we please rename `gps_hdop` and `gps_vdop` to `dilution_of_precision` and set the *type instance* to `horizontal` and `vertical`?